presentation-compiler/src/main/dotty/tools/pc/SignatureHelpProvider.scala

```diff
@@ -70,7 +70,7 @@ object SignatureHelpProvider:
         new l.SignatureHelp(
           signatureInfos.map(signatureToSignatureInformation).asJava,
           callableN,
-          paramN
+          if signatureInfos.isEmpty then null else paramN
```

Comment:
Still need to decide wether we use -1 or null

Reply:
I didn't see this. I've answered below